src/mappings/shared/jb721DelegateToken.ts

```diff
@@ -0,0 +1,75 @@
+import { Address, dataSource, log } from "@graphprotocol/graph-ts";
+
+import { JB721DelegateToken, Participant } from "../../../generated/schema";
+import { Transfer } from "../../../generated/templates/ERC20/ERC20";
+import { ERC721 } from "../../../generated/templates/JB721DelegateToken/ERC721";
+import { newParticipant } from "../../utils/entity";
+import {
+  idForJB721DelegateToken,
+  idForParticipant,
+  idForProject,
+} from "../../utils/ids";
+
+export function handleTransfer(event: Transfer): void {
+  const context = dataSource.context();
+  const projectId = context.getBigInt("projectId");
+  const pv = context.getString("pv");
+  const address = dataSource.address();
+  const contract = ERC721.bind(Address.fromBytes(address));
+
+  const id = idForJB721DelegateToken(Address.fromBytes(address));
```

Comment:
I think this should be a concatenation of address + token ID? e.g.
```jsx
const id = `${idForJB721DelegateToken(Address.fromBytes(address))-${event.params.tokenId}}`;
```
otherwise we only get one instance of `JB721DelegateToken` per contract rather than per token